pkg/datapath/linux/bandwidth/cell_darwin.go

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+//go:build darwin
+
+package bandwidth
+
+// NOTE: We need to mock out the whole bandwidth package on Darwin, because importing
+//       the bwmap package will cause an import of pkg/ebpf and pkg/bpf which don't
+//       build on darwin.
+
+type Manager interface {
```

Comment:
Perhaps better to add e.g. `types.go` where the `Manager` interface lives so these don't need to be kept in sync?